config/v1/zz_generated.crd-manifests/0000_10_config-operator_01_nodes-CustomNoUpgrade.crd.yaml

```diff
@@ -53,6 +53,10 @@ spec:
                 - v2
                 - ""
                 type: string
+                x-kubernetes-validations:
+                - message: cgroups "v1" is not supported, please update to cgroups
```

Comment:
so this API field is cgroupMode, but we refer to it solely here in plural. I think it should be
```
cgroup "v1" is not supported, please update to cgroup "v2"
```

or even replace cgroup with cgroupMode...

Reply:
Addressed